respond.src.js

```diff
@@ -267,7 +267,11 @@ window.matchMedia = window.matchMedia || (function( doc, undefined ) {
 			}
 			
 			//inject active styles, grouped by media type
+			var keys = [];
 			for( var k in styleBlocks ){
+				keys.push( k );
```

Comment:
Yeah I wondered about that when I was implementing it, but wanted to make the smallest change that seemed to work.  I've implemented your suggestion.